### DISCUSSION TITLE
Laravel Taxify package, to simplify tax (VAT) calculations in Laravel

### DISCUSSION THREAD

**Discussion by WatheqAlshowaiter** [HassanZahirnia/laravel-package-ocean#182] [Package Suggestions]
[Package link](https://github.com/omaralalwi/laravel-taxify)
[Docs](https://omaralalwi.github.io/laravel-taxify/)

## Features

- Calculate tax for individual amounts or a collection of amounts
- Retrieve tax amount, rate, and type for any profile as individual.
- Retrieve tax `amount_with_tax` and `tax_amount`  and `tax_rate` for any profile as individual for one amount or a collection of amounts
- Get tax rate as a percentage (for percentage type only)
- Customizable configuration options through environment variables easy.
- helper functions easy-to-use.
- support all php and laravel versions.
- Exception Handling: Robust error handling to ensure smooth operation and easy debugging.
- Logging: Automatic logging of errors and exceptions for better error tracking and debugging.
- Unit Tests.

**Comment by HassanZahirnia** [comment]
Hi there!

I just had a look at your package and it seems it does not require Laravel as a dependency (in the composer.json). But you advertised it as a Laravel package. Could it be it's a general php utility composer, and not just specific to Laravel?

Because within Laravel Package Ocean database, we would extract a list of Laravel dependency for Laravel specific packages, so we know what's the compatibility version range.

**Reply by omaralalwi** [reply]
yes it is did not depend on laravel framework in it's composer file, but still need to some updates to make it working in any PHP app. (we are working in these updates) , and will push some updates in coming days.
when we push that updates , will let you know.

thank you for your response .
